de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties

```diff
@@ -906,3 +906,5 @@ create_and_link_type_webpage=Webpage
 claim_publications_by=Claim publications by
 claim_publications_by_doi=DOI
 claim_publications_by_pmid=PubMed ID
+
+term_capitalized = Begriff
```

Comment:
I am not seeing `group_capitalized` defined in anywhere??

Reply:
group_capitalized already exists in Vitro-languages (for example https://github.com/vivo-project/Vitro-languages/blob/sprint-i18n/de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties).
Do you want me to add that extra one for the Capability Map?

Reply:
Thanks. I was not seeing it.
As a separate note, I wonder why "group" is not actually capitalized in:
https://github.com/vivo-project/Vitro-languages/blob/sprint-i18n/de_DE/webapp/src/main/webapp/i18n/all_de_DE.properties#L877